src/bot.js

```diff
@@ -22,6 +22,17 @@ const prefix = '+';
 // Create new Parser
 const parser = new Parser();
 
+function sorter(f) {
```

Comment:
I don't like calling it 'sorter'. perhaps parser or formatter would be better

Reply:
Will update the name